ForumProject/notifications/models.py

```diff
@@ -0,0 +1,30 @@
+from django.db import models
+from projects.models import Project
+from investors.models import Investor
+from startups.models import Startup
+
+class Notification(models.Model):
+    
+    TRIGGER_CHOICES = [
+        ('follower(s) list changed', 'follower(s) list changed'),
+        ('subscription changed', 'subscription changed'),
+        ('project status changed', 'project status changed'),
+        ('project budget changed', 'project budget changed'),
+        ('message sent', 'message sent'),
+    ]
+    INITIATOR_CHOICES = [
+        ('investor', 'investor'),
+        ('project', 'project'),
+    ]    
+    project = models.ForeignKey(Project, on_delete=models.SET_NULL, related_name='notice_on_project')
+    startup_name = models.CharField(max_length=Startup._meta.get_field('startup_name').max_length)
+    investor = models.ForeignKey(Investor, on_delete=models.SET_NULL, related_name='notice_to_investor')
+    trigger = models.CharField(max_length=50, choices=TRIGGER_CHOICES)
+    initiator = models.CharField(max_length=8, choices=INITIATOR_CHOICES)
+    date_time = models.DateTimeField(auto_now_add=True)
+
+    def __str__(self):
+        addressee = f'Startup {self.startup_name}' if self.initiator == 'investor' else f'Investor {self.investor.investor_name}'
+        return f'Notification of {self.date_time} to {addressee} on Project {self.project.project_name}'
```

Comment:
Since project and investor can be `None`, your `__str__` method could raise an `AttributeError` if these objects are null and you try to access `project.name` or `investor.investor_name`. You should add checks to handle these cases gracefully